app/controllers/mails_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MailsController < ApplicationController
+  before_action :admin_required
+
+  def write_email
+    @simplemail = SimpleMail.new 
+    @simplemail.sender = current_user.email
+    @simplemail.addressee = Person.find(params[:id]).email
+  end
+
+  def send_email
+    @simplemail = SimpleMail.new(mail_params)
```

Comment:
Tu wyskakuje błąd. Obecnie wyświetla się już formularz wysyłania wiadomości, ale po kliknięciu "wyślij" pojawia się informacja "undefined local variable or method `mail_params' for #<MailsController:0x007f42f27449d8>"